source/features/tag-changelog-link.tsx

```diff
@@ -87,27 +90,22 @@ async function init(): Promise<void> {
 
 	for (const [index, container] of allTags.entries()) {
 		const previousTag = getPreviousTag(index, allTags);
+		if (!previousTag) {
+			return;
+		}
 
-		if (previousTag) {
-			// Signed releases include on mobile include a "Verified" <details> inside the `ul`. `li:last-of-type` excludes it.
-			// Example: https://github.com/tensorflow/tensorflow/releases?after=v1.12.0-rc1
-			for (const lastLink of select.all('.list-style-none > li:last-of-type', container.element)) {
-				lastLink.after(
-					<li className={lastLink.className}>
-						<a
-							className="muted-link tooltipped tooltipped-n"
-							aria-label={'See changes since ' + decodeURIComponent(previousTag)}
-							href={buildRepoURL(`compare/${previousTag}...${allTags[index].tag}`)}
-						>
-							<DiffIcon/> Changelog
-						</a>
-					</li>
-				);
-
-				// `lastLink` is no longer the last link, so it shouldn't push our new link away.
-				// Same page as before: https://github.com/tensorflow/tensorflow/releases?after=v1.12.0-rc1
-				lastLink.classList.remove('flex-auto');
-			}
+		for (const lastLink of select.all('.list-style-none > .d-block:nth-child(2), .list-style-none > .d-inline-block:last-child', container.element)) {
```

Comment:
Can you comment each selector?
```suggestion
		const lastLinks = select.all([
			'.list-style-none > .d-block:nth-child(2)', // XXX
			'.list-style-none > .d-inline-block:last-child' // YYY
		], container.element);
		for (const lastLink of lastLinks) {
```